openapi/openapi.yaml

```diff
@@ -713,7 +729,12 @@ components:
         - WDL
         - NFL
         - GALAXY
-        - SMK
+        - SMK        
+    DescriptorTypeVersion:
+      type: string
+      description: The language version for a given descriptor type. The version should correspond
+        to the actual declared version of the descriptor. For example, tools defined in CWL may have a version of `v1.0.2`
```

Comment:
> For example, tools defined in CWL may have a version of `v1.0.2`

This reads a little prescriptive to me, e.g., they may not have a version v1.0.0. I suggest "could" instead of "may"